src/sentry/static/sentry/app/components/releaseProjectStatSparkline.jsx

```diff
@@ -19,11 +22,13 @@ const ReleaseProjectStatSparkline = React.createClass({
       loading: true,
       error: false,
       stats: [],
+      newIssueCount: 0,
```

Comment:
since the loading state doesn't appear to take this request into consideration, maybe you could start off with this being null and only show that part of the UI once you actually have a value? Another option would be to stay in the loading state until both requests finish. I'd just be nice to avoid a possible flash of `0` before we get the actual value...